src/models/geometric_models.jl

```diff
@@ -241,22 +241,26 @@ end
 A delta parabolic wisp in the image domain.
 The wisp is a parabolic line segment centered at zero, with roots ±1 and a yintercept of 1.
 """
-struct Wisp{F} <: GeometricModel end
-Wisp() = Wisp{Float64}()
-function Wisp(a, h) end
+struct UnitWisp{F} <: GeometricModel end
+UnitWisp() = UnitWisp{Float64}()
+function UnitWisp(a, h) end
```

Comment:
for the UnitWisp(a,h) method we have two options. We can either delete the method or build a convenience function for a stretched wisp, something like

```julia
UnitWisp(a,h) = stretched(UnitWisp(), a, h)
``` 

although at that point we should probably call it Wisp right?